pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix

```diff
@@ -190,7 +190,9 @@ stdenv.mkDerivation rec {
       "''${gappsWrapperArgs[@]}" \
       "''${qtWrapperArgs[@]}" \
       --prefix LD_LIBRARY_PATH : "${xorg.libXcursor}/lib" \
-      --suffix PATH : ${lib.makeBinPath [ xdg-utils ]}
+      --suffix PATH : ${lib.makeBinPath [ xdg-utils ]} \
+      --unset  QT_SCREEN_SCALE_FACTORS \
```

Comment:
Add a comment to explain why we need unset `QT_SCREEN_SCALE_FACTORS`


There doesn't seem to be a better solution until this bug is fixed upstream

Reply:
The back ticks around comments is a trick from https://stackoverflow.com/a/12797512